docs/hugo/content/tools/asoctl.md

```diff
@@ -29,7 +29,37 @@ Available Commands:
 
 ## Clean CRDs
 
-TBC
+This command can be used to prepare ASOv2 `v1alpha1api`(deprecated in v2.0.0) CustomResources and CustomResourceDefinitions for ASO `v2.0.0` release. 
+It ensures that any ASOv2 `v1alpha1api` deprecated version resources that may have been stored in etcd get migrated to `v1beta` version before upgrading ASO to `v2.0.0`. 
+
+```bash
+$ asoctl clean crds --help
+Clean deprecated CRD versions from cluster
+
+Usage:
+  asoctl clean crds [flags]
+
+Flags:
+      --dry-run   
+  -h, --help      help for clean
+
+Global Flags:
+      --verbose   Enable verbose logging
+```
+
+`--dry-run` flag outputs about CRDs and CRs to be updated and **does not** modify any CRD and CRs.
+
+### Steps for migration using `asoctl clean crds`:
+
+- Ensure the current ASOv2 version is `beta.5`
+- Run `asoctl clean crds`
+- Upgrade ASOv2 to `2.0.0`
```

Comment:
Readers may not understand that you're showing a command line.

```suggestion
**Prerequisite:** Ensure the current ASO v2 version in your cluster is `beta.5`.

Run the migration tool:
``` bash
$ asoctl clean crds
```

Once that's successfully run, you can upgrade ASO to `v2.0.0`
```

Maybe include some sample output for the command?

Reply:
Done!